aws-datastore/src/main/java/com/amplifyframework/datastore/model/SerializedModelHelper.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.datastore.model;
+
+import com.amplifyframework.AmplifyException;
+import com.amplifyframework.core.Amplify;
+import com.amplifyframework.core.model.Model;
+import com.amplifyframework.core.model.ModelField;
+import com.amplifyframework.datastore.DataStoreException;
+import com.amplifyframework.datastore.appsync.SerializedModel;
+import com.amplifyframework.logging.Logger;
+
+import java.util.Map;
+
+/**
+ * Helpers class that contains {@link SerializedModel} related utilities.
+ */
+public final class SerializedModelHelper {
+    private static final Logger LOGGER = Amplify.Logging.forNamespace("amplify:aws-datastore");
+
+    private SerializedModelHelper() {
+        // contains only static helper methods and should not be instantiated
+    }
+
+    /**
+     * Uses seriazlied data to return the field value.
+     *
+     * @param model the model instance which contains serialized data
+     * @param field the model field
+     * @param <M> the model concrete type
+     * @return the field value or <code>null</code>
+     * @throws DataStoreException in case of a error happens during the dynamic reflection calls
+     */
+    public static <M extends Model> Object getValue(M model, ModelField field) throws DataStoreException {
```

Comment:
Could you just make this a simple method on the `SerializedModel` class?  
```
public Object getValue(ModelField field) {
      return getSerializedData().get(field.getName());
}
```

Reply:
yes, updated `SerializedModel` and deleted this class.